CsvParser/csvParser_test.go

```diff
@@ -104,29 +164,141 @@ func Test_CsvParserFiftyFirstEntry(t *testing.T) {
 	}
 }
 
-func Test_MakeJsonObjFile(t *testing.T) {
+func Test_FileHierachy(t *testing.T) {
+	ex, err := os.Executable()
+	if err != nil {
+		panic(err)
+	}
+	originPath := filepath.Dir(ex)
+
+	incidentMap := mapIncidentsToLocAndISP(generateTestIncidents())
 
-	testIncidentArr := make([]incident.Incident, 1, 1)
-	incidents := CsvParser("incidentfile.csv")
-	testIncident := convertDefaultIncidentToIncident(incidents)
-	testIncidentArr[0] = testIncident[0]
+	locationCodesArr := make([]string, 0)
+	locationCodesArr = append(locationCodesArr, "/eu/fr")
+	locationCodesArr = append(locationCodesArr, "/eu/fr/AS10774x.json")
+	locationCodesArr = append(locationCodesArr, "/eu/fr/AS10774z.json")
+	locationCodesArr = append(locationCodesArr, "/na/us")
 
+	tests := []struct {
+		name        string
+		input  []string
+	}{
+		{
+			name: "Check of the directories are being created",
```

Comment:
Typo here: Check "if" the directories are being created

Reply:
Good job catching something like this wow